index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome Section</a>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Welcome to our Portfolio Webpage</h1>
+    </section>
+
+    <section id="projects">
+      <h3 class = "project-tile">
+      <a href = "#">Project 01</a>
```

Comment:
- [ ] It is HTML/CSS best practice to use space-less equal signs.
Use `class="project-tile"`, not `class = "project-tile"`